importers/bano/README.md

```diff
@@ -2,40 +2,22 @@
 
 This importer import the addresses from [BANO]. It requires as first argument, the csv file (you can
 download it from [here](http://bano.openstreetmap.fr/data/)) in which it'll find the data from
-[BANO] and runs as follows:
+[BANO].
 
-```bash
-$ cargo run --release -- [BANO csv file]
-```
+## How it works
 
-The generated database has two tables which look like this:
+The script reads the provided CSV file. Each line is then added if it has the following elements:
 
-```sql
-CREATE TABLE IF NOT EXISTS addresses(
-    lat REAL NOT NULL,
-    lon REAL NOT NULL,
-    number TEXT,
-    street TEXT NOT NULL,
-    unit TEXT,
-    city TEXT,
-    district TEXT,
-    region TEXT,
-    postcode TEXT,
-    PRIMARY KEY (lat, lon, number, street, city)
-);
+ * longitude
+ * latitude
+ * street name
+ * house number
 
-CREATE TABLE IF NOT EXISTS addresses_errors(
-    lat REAL,
-    lon REAL,
-    number TEXT,
-    street TEXT,
-    unit TEXT,
-    city TEXT,
-    district TEXT,
-    region TEXT,
-    postcode TEXT,
-    kind TEXT
-);
+## Running it
+
+You can run it like this:
+```bash
+$ cargo run --release -- [BANO csv file]
 ```
 
 ## Using it as a library
```

Comment:
`import_addresses` actually has the following skeleton:

```rust
fn import_addresses<P: AsRef<Path>, T: CompatibleDB>(file_path: P, db: &mut T);
```

The skeleton is the wrong one and `file_path` is called `file` in arguments documentation.